src/modules/exchange/shapeshift.js

```diff
@@ -51,6 +51,21 @@ export function makeShapeshiftApi (ai: ApiInput) {
       return +json.rate
     },
 
+    async getExchangeSwapInfo (
+      fromCurrency: string,
+      toCurrency: string
+    ): Promise<{
+      pair: string,
+      rate: number,
+      limit: number,
+      min: number,
+      minerFee: number
+    }> {
+      const pair = `${fromCurrency}_${toCurrency}`
+      const json = await api.get(`/marketinfo/${pair}`)
+      return json
```

Comment:
Assuming the format uses nativeAmounts, we need to process the JSON into strings.